lti_consumer/lti_1p3/consumer.py

```diff
@@ -90,20 +93,30 @@ def _get_user_roles(role):
 
     def prepare_preflight_url(
             self,
-            hint="hint",
-            lti_hint="lti_hint"
+            launch_data,
     ):
         """
         Generates OIDC url with parameters
         """
+        user_id = launch_data.user_id
+
+        # Set the launch_data in the cache. We store the launch_data in the cache instead of sending a representation of
```

Comment:
kind of a NIT? IMO I don't feel there's a lot of value in explaining a solution we don't implement with the query param. I think it would be more useful to just explain why we store this (to reference later) instead.